src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/IssueDocumentHandlerTest.java

```diff
@@ -147,4 +148,62 @@ public void testDocumentPayloadValues() {
         assertEquals(localDate, payload.getGeneratedDate());
         assertEquals("Barry Allen", payload.getIdamSurname());
     }
+
+    @Test
+    public void givenSetAsideState_thenReturnSetAsideDecisionNotice() {
+        SscsCaseData sscsCaseData = SscsCaseData.builder()
+            .ccdCaseId("1")
+            .postHearing(PostHearing.builder()
+                .setAside(SetAside.builder()
+                    .action(SetAsideActions.GRANT)
+                    .build())
+                .build())
+            .build();
+
+        String documentTypeLabel = new IssueDocumentHandler().getDocumentTypeLabel(sscsCaseData, DocumentType.DECISION_NOTICE, "adfdsf2");
+
+        assertEquals("Set Aside Decision Notice", documentTypeLabel);
+    }
+
+    @Test
+    public void givenSetAsideStateRefused_thenReturnSetAsideDecisionNotice() {
+        SscsCaseData sscsCaseData = SscsCaseData.builder()
+            .ccdCaseId("1")
+            .postHearing(PostHearing.builder()
+                .setAside(SetAside.builder()
+                    .action(SetAsideActions.REFUSE)
+                    .build())
+                .build())
+            .build();
+
+        String documentTypeLabel = new IssueDocumentHandler().getDocumentTypeLabel(sscsCaseData, DocumentType.DECISION_NOTICE, "adfdsf2");
+
+        assertEquals("Set Aside Decision Notice", documentTypeLabel);
+    }
```

Comment:
Combine into a ParameterizedTest with EnumSource of SetAsideActions

Reply:
I have tried to use the ParameterizedTest with the Enum Source, in this case 2 of the test case will fail for following enum values (ISSUE_DIRECTIONS, REFUSE_SOR).
Due to this I have created a separate the JUNIT test case.

Reply:
Updated to use the ParameterizedTest  test.